doc/modules/decomposition.rst

```diff
@@ -933,9 +926,8 @@ stored components::
       <http://www.bsp.brain.riken.jp/publications/2009/Cichocki-Phan-IEICE_col.pdf>`_
       A. Cichocki, A. Phan, 2009
 
-    .. [6] `"Algorithms for nonnegative matrix factorization with the beta-divergence"
-      <https://arxiv.org/pdf/1010.1763.pdf>`_
-      C. Fevotte, J. Idier, 2011
+    .. [6] :arxiv:`C. Fevotte, J. Idier, 2011. "Algorithms for nonnegative matrix factorization with
+           the beta-divergence" <1010.1763>`
```

Comment:
Let's be consistent with the citation style of the surrounding reference box (even if it's not consistent with the citation style of the whole project).

```suggestion
    .. [6] :arxiv:`"Algorithms for nonnegative matrix factorization with
           the beta-divergence" <1010.1763>` C. Fevotte, J. Idier, 2011.
```